### DISCUSSION TITLE
Welcome to Shortcutomation!

### DISCUSSION THREAD

**Discussion by huaminghuangtw** [huaminghuangtw/Apple-Shortcuts-Gallery#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chvmnaveen** [comment]
It would be great if you create a tutorial for how to use one of the Shortcut. Because when I tried to download and access a Shortcut on my phone, I'm not able to finish it as there is no idea what is missing.

**Reply by huaminghuangtw** [reply]
@chvmnaveen  Hi, it would be really helpful if you could provide more details about the issues you’re facing when trying to download and access a Shortcut on your phone. Could you please share your experience and any error messages you encountered? This will help me understand the problem better and create a more effective tutorial.🙂

**Reply by huaminghuangtw** [reply]
@chvmnaveen BTW, to use this Shortcuts library without any dependency issues, one needs to install the following tools:

- [Data Jar](https://apps.apple.com/app/id1453273600)
- [Actions](https://apps.apple.com/app/id1586435171)
- [Toolbox Pro](https://apps.apple.com/app/id1476205977)

But I would suggest to first try those simple Shortcuts in the "[Standalone Fun](https://github.com/huaminghuangtw/Apple-Shortcuts-Gallery/tree/main/All%20Shortcuts/Standalone%20Fun)" folder, as they can be run without relying on other Shortcuts/tools.
